nsxt/resource_nsxt_policy_group.go

```diff
@@ -844,12 +855,18 @@ func resourceNsxtPolicyGroupCreate(d *schema.ResourceData, m interface{}) error
 	description := d.Get("description").(string)
 	tags := getPolicyTagsFromSchema(d)
 
+	var groupTypes []string
+	groupType := d.Get("group_type").(string)
+	if groupType != "" && nsxVersionHigherOrEqual("3.2.0") {
```

Comment:
I don't think this version check would work with NSX version <3.2, since empty list of group_types would still be sent to the platform, and NSX would not recognize this property. The version check needs to define whether `GroupType` is set in the struct.